package first;

public class ExceptionHandling 
{
	//throws ==> our own created exceptions
	public static void main(String[] args) throws ArithmeticException
	{
			for (int i=5; i>=-5;i--) 
			{
				System.out.println(10/i);
			}
	}
}
